h2o-r/h2o-package/R/models.R

```diff
@@ -3064,5 +3064,46 @@ h2o.deepfeatures <- function(object, data, layer) {
   h2o.getFrame(dest_key)
 }
 
+#'
+#' The H2OTree class.
+#'
+#' This class represents a model of a Tree built by one of H2O's algorithms (GBM, Random Forest).
+#' @slot left_children An \code{integer} vector with left children of tree's nodes
+#' @slot right_children An \code{integer} vector with right children of tree's nodes
+#' @slot descriptions A \code{character} vector with descriptions for each node to be found in the tree.
+#' @slot model_name A \code{character} string with the name of the model this tree has been generated in.
```

Comment:
We use "algorithm" elsewhere in the R code for this same string, so it might make sense to re-use that term here.

```
> fit <- h2o.gbm(y = 5, training_frame = as.h2o(iris))
  |=======================================================================================| 100%
  |=======================================================================================| 100%
> fit@algorithm
[1] "gbm"
```

Reply:
It works for both GBM and DRF. I added a line into the getTreeModel documentation that says exactly that. The demonstration still uses GBM.

Reply:
Also added R test that demonstrates fetching DRF models.

Reply:
Ok, in this case, then I'd suggest that we call the slot `"algorithm"` to match the existing naming convention.